src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataDeserializerContext.cs

```diff
@@ -47,6 +47,11 @@ public partial class ODataDeserializerContext
         /// </summary>
         internal IWebApiUrlHelper InternalUrlHelper { get; private set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether request body binding should be case insensitive.
+        /// </summary>
+        internal bool EnableCaseInsensitiveModelBinding { get; set; }
```

Comment:
It is a bit strange to have a different setting for controlling case sensitivity for properties in a payload versus properties in the URL.

Reply:
I tested and found that properties in the URL are case-insensitive by default

Reply:
yes; that's what makes me think we should consider fixing this as a bug and making it the default (which can be disabled through a compatibility option) rather than providing a separate method to opt-in.

At least in 8.0 we should strongly consider just making this the default and not adding a separate flag for controlling case insensitivity in the body.

---
In reply to: [503683409](https://github.com/OData/WebApi/pull/2311#discussion_r503683409) [](ancestors = 503683409)

Reply:
#Resolved

Reply:
Can you create an issue, assigned to 8.0 and marked "breaking change opportunity" to remove this flag?

---
In reply to: [505930201](https://github.com/OData/WebApi/pull/2311#discussion_r505930201) [](ancestors = 505930201,503683409)